portfolio/src/main/webapp/about.html

```diff
@@ -1 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope|Nanum+Pen+Script&display=swap" rel="stylesheet">
+  </head>
+  <body>
+  <div class="nav">
+    <a class="active" href="index.html">Home</a>
+    <a href="aboutme.html">Translate</a>
+    <a href="about.html">About Me</a>
+  </div>
+    <div id="content">
+    <div style="background-color:white;background-image:url(//www.html.am/images/backgrounds/background-image-2.gif);background-repeat:repeat;"></div>
+      <h1>About Me!</h1>
+      <p>Hello! My name is Jazmin Vences.</p>
+      <br>
+      <p>If you get to know me, one thing that is apparent is my love for animals. My family often says I was probably a zookeeper in my past life because of the amount of animals I've had.</p>
+      <br>
+      <div id="slider">
+	    <figure>
```

Comment:
Minor nit: try to keep indentation consistent and have child elements indented 2 more spaces than the parent.